src/components/Controls/Controls.css.ts

```diff
@@ -90,10 +90,10 @@ const flash = keyframes`
 `;
 
 export const FlashingControlButton = styled(ControlButton)<{
-  animate: boolean;
+  $animate: boolean;
```

Comment:
Using the ﻿$ symbol to change this to a transient prop to stop React from complaining about it